server/project-service/src/main/java/edu/illinois/ncsa/incore/service/project/utils/CommonUtil.java

```diff
@@ -0,0 +1,174 @@
+                break;
+            case "inventorytype":
+                comparator = Comparator.comparing(DFR3MappingResource::getInventoryType, Comparator.nullsLast(String::compareToIgnoreCase));
+                break;
+            case "type":
+                comparator = Comparator.comparing(DFR3MappingResource::getType);
+                break;
+            case "id":
+                comparator = Comparator.comparing(DFR3MappingResource::getId);
+                break;
+            case "name":
+                comparator = Comparator.comparing(DFR3MappingResource::getName);
+                break;
+            case "date":
+            default:
+                comparator = Comparator.comparing(DFR3MappingResource::getDate);
+                break;
+        }
+
+        if ("desc".equalsIgnoreCase(order)) {
+            comparator = comparator.reversed();
+        }
+
+        return comparator;
+    }
+
+    public static Comparator<WorkflowResource> workflowComparator(String sortBy, String order) {
+        Comparator<WorkflowResource> comparator;
+
+        switch (sortBy.toLowerCase()) {
+            case "type":
+                comparator = Comparator.comparing(WorkflowResource::getType);
+                break;
+            case "id":
+                comparator = Comparator.comparing(WorkflowResource::getId);
+                break;
+            case "title":
+                comparator = Comparator.comparing(WorkflowResource::getTitle, Comparator.nullsLast(String::compareToIgnoreCase));
+                break;
+            case "created":
+            default:
+                comparator = Comparator.comparing(WorkflowResource::getCreated, Comparator.nullsLast(Comparator.naturalOrder()));
+                break;
+        }
+
+        if ("desc".equalsIgnoreCase(order)) {
+            comparator = comparator.reversed();
+        }
+
+        return comparator;
+    }
+
+    public static Comparator<VisualizationResource> visualizationComparator(String sortBy, String order) {
+        Comparator<VisualizationResource> comparator;
+
+        switch (sortBy.toLowerCase()) {
+            case "type":
+                comparator = Comparator.comparing(VisualizationResource::getType);
+                break;
+            case "id":
+                comparator = Comparator.comparing(VisualizationResource::getId, Comparator.nullsLast(String::compareToIgnoreCase));
+                break;
+            case "name":
+                comparator = Comparator.comparing(VisualizationResource::getName, Comparator.nullsLast(String::compareToIgnoreCase));
+                break;
+            case "date":
+            default:
+                comparator = Comparator.comparing(VisualizationResource::getDate, Comparator.nullsLast(Comparator.naturalOrder()));
+                break;
+        }
+
+        if ("desc".equalsIgnoreCase(order)) {
+            comparator = comparator.reversed();
+        }
+
+        return comparator;
+    }
+
+
```

Comment:
Can you remove the extra line break here?

Reply:
removed